src/v/kafka/server/handlers/alter_partition_reassignments.cc

```diff
@@ -64,59 +64,72 @@ partitions_request_iterator validate_replicas(
     return valid_range_end;
 }
 
+/*
+ * begin          - starting position for a vector<reassignable_partition>
+ * end            - stopping position for a vector<reassignable_partition>
+ * resp_it        - a wrapper to std::back_inserter
+ * topic_response - a reassignable_topic_response to put errors into
+ * alive_nodes    - list of RP nodes that are live
+ * tp_metadata    - topic metadata used to check replication factor
+ *
+ * return:  an iterator that represents the stop position of all valid
+ * partitions
+ */
```

Comment:
nit: I've noticed that there isn't a lot of documentation in the code, but for areas that are documented, javadoc tends to be the preferred style:

```suggestion
/**
 * @brief Validates partitions and places invalid partitions into @p resp_it
 * @param begin starting position for a vector<reassignable_partition>
 * @param end stopping position for a vector<reassignable_partition>
 * @param resp_it  a wrapper to std::back_inserter
 * @param topic_response a reassignable_topic_response to put errors into
 * @param alive_nodes list of RP nodes that are live
 * @param tp_metadata topic metadata used to check replication factor
 *
 * @return an iterator that represents the stop position of all valid
 * partitions
 */
```

Reply:
This is not necessarily true, our commenting styles are inconsistent in the tree.

Reply:
done.

Reply:
No doubt.  But maybe we can lead the charge towards consistency :horse_racing: